featuretools/primitives/standard/natural_language_primitives/api.py

```diff
@@ -0,0 +1,43 @@
+# flake8: noqa
+from featuretools.primitives.standard.natural_language_primitives.count_string import (
+    CountString,
+)
+from featuretools.primitives.standard.natural_language_primitives.mean_characters_per_word import (
+    MeanCharactersPerWord,
+)
+from featuretools.primitives.standard.natural_language_primitives.median_word_length import (
+    MedianWordLength,
+)
+from featuretools.primitives.standard.natural_language_primitives.num_unique_separators import (
+    NumUniqueSeparators,
+)
+from featuretools.primitives.standard.natural_language_primitives.number_of_common_words import (
+    NumberOfCommonWords,
+)
+from featuretools.primitives.standard.natural_language_primitives.number_of_hashtags import (
+    NumberOfHashtags,
+)
+from featuretools.primitives.standard.natural_language_primitives.number_of_mentions import (
+    NumberOfMentions,
+)
+from featuretools.primitives.standard.natural_language_primitives.number_of_unique_words import (
+    NumberOfUniqueWords,
+)
+from featuretools.primitives.standard.natural_language_primitives.number_of_words_in_quotes import (
+    NumberOfWordsInQuotes,
+)
+from featuretools.primitives.standard.natural_language_primitives.punctuation_count import (
+    PunctuationCount,
+)
+from featuretools.primitives.standard.natural_language_primitives.title_word_count import (
+    TitleWordCount,
+)
+from featuretools.primitives.standard.natural_language_primitives.total_word_length import (
+    TotalWordLength,
+)
+from featuretools.primitives.standard.natural_language_primitives.upper_case_count import (
+    UpperCaseCount,
+)
+from featuretools.primitives.standard.natural_language_primitives.whitespace_count import (
+    WhitespaceCount,
+)
```

Comment:
A couple of minor things since we are going into the restructure directory route, to reduce the lines here we can just use relative imports, it's a bit cleaner. I would also move all of this code to `__init__.py` since we're trying to remove api.py as it's redundant to how __init__.py works. 

ex: https://github.com/alteryx/featuretools/pull/2187/files#diff-4ffbd94fdc40f8ee9f37abe2ead42e08aca5156d0b13c958a07573691929c9ce

Reply:
Yeah, I'll clean this up in the PR for issue #2179